test/e2e_node/e2e_service.go

```diff
@@ -190,7 +190,9 @@ func (es *e2eService) startEtcd() (*killCmd, error) {
 		}
 		etcdPath = defaultEtcdPath
 	}
-	cmd := exec.Command(etcdPath)
```

Comment:
Aha!  I'm not a CoreOS user, typically.

Should we extend the pattern to look for `etcd3`?

(This entire PR was triggered by me installing etcd V3 without realising this would cause everything to fail)